content/docs/machine-learning/creating-a-new-kubeflow-notebook.md

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 4
+title: Creating a new Kubeflow notebook
+description: Creating a new notebook in Civo's machine learning Kubeflow service
+---
+
+<head>
+  <title>Creating a new Kubeflow notebook | Civo Documentation</title>
+</head>
+
+To create a new Kubeflow notebook, you will need to provide a name, select a development environment, and make a few architectural decisions. All of these are explained below.
```

Comment:
Do we in general want to do an introduction to Kubeflow stuff?

So something like have this diagram and explain that when we talk about notebook servers in Kubeflow

![image](https://github.com/civo/docs/assets/39672672/c6d893b0-698d-4ab9-9cc1-598dc966e3a1)

Maybe something like:

> Whenever you create a new Kubeflow Notebook server you are essentially creating a Kubernetes StatefulSet. And any code you run in the Notebook server run son Kubernetes pods.

> A StatefulSet is a controller helps you deploy and scale groups of Kubernetes pods. The StatefulSet controller also instantiates and manages PersistentVolumeClaims for each Pod instance which we will take a better look at in the next section. A StatefulSet is commonly used when you want to run several pods of the same type in parallel, but each of the pods is required to have a specific identity.

Reply:
Yes!!!!

Reply:
Note that these docs were create by me and kai - we're not specialists in ML so there's definitely a lot of improvement to be made and section to be explained in more depth but still accessible to beginners like us :) Feel free to edit the docs as you see fit and add us as reviewers

Reply:
Thanks, I was talking with @kaihoffman, and he shared about the goal of these docs and the style they need to be written in: not too much technical details, not too much Kubeflow, not too much Kubernetes, just how to use Kfaas.